base/sparse/cholmod.jl

```diff
@@ -89,10 +88,9 @@ function __init__()
                 of CHOLMOD, or download the generic binaries
                 from www.julialang.org, which ship with the correct
                 versions of all dependencies.
-            """)
```

Comment:
Why delete the empty lines and add these spaces? Not super important, but I think the previous behavior is nicer.